### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**

### CODE REVIEW

**Comment by kianmeng** [comment]
@josevalim Any updates on this?

File: lib/phoenix/pubsub.ex [hunk 1]
```diff
@@ -84,7 +84,7 @@ defmodule Phoenix.PubSub do
   ## Options
 
     * `:name` - the name of the pubsub to be started
-    * `:adapter` - the adapter to use (defauls to `Phoenix.PubSub.PG2`)
+    * `:adapter` - the adapter to use (default to `Phoenix.PubSub.PG2`)
```
Comment (josevalim) [inline lib/phoenix/pubsub.ex:87]:
```suggestion
    * `:adapter` - the adapter to use (defaults to `Phoenix.PubSub.PG2`)
```

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Comment by kianmeng** [comment]
:+1: :tada:

**Pull request merged**